packages/node_modules/@webex/plugin-meetings/src/locus-info/selfUtils.js

```diff
@@ -244,7 +244,7 @@ SelfUtils.mutedByOthers = (oldSelf = {}, changedSelf) => {
 SelfUtils.getSipUrl = (partner, type, sipUri) => {
   // For webex meeting the sipUrl gets updated in info parser
   if (partner && type === _CALL_) {
-    return {sipUri: partner.person.sipUrl || partner.person.emailAddress || partner.person.email};
+    return {sipUri: partner.person.sipUrl};
```

Comment:
As @arun3528 mentioned in the space, let's change this to user ID for 1:1 calls.

Reply:
We'll need to update the tests for this as well.